pkg/component/shared/istio.go

```diff
@@ -239,6 +259,37 @@ func IsZonalIstioExtension(labels map[string]string) (bool, string) {
 	return false, ""
 }
 
+// ShouldEnforceSpreadAcrossHosts checks whether all given zones have at least two nodes so that Istio can be spread across hosts in each zone
```

Comment:
```suggestion
// ShouldEnforceSpreadAcrossHosts checks whether all given zones have at least two nodes so that Istio can be spread across hosts in each zone.
```

Reply:
Good catch.